src/components/modules/dragNDrop.ts

```diff
@@ -118,5 +118,10 @@ export default class DragNDrop extends Module {
    */
   private processDragOver(dragEvent: DragEvent): void {
     dragEvent.preventDefault();
+    const selectedBlock = this.Editor.BlockManager.currentBlock;
+    if (selectedBlock.holder.querySelector('img')) {
```

Comment:
It is the hardcoding of the Image Tool internal thing into editor core. I think we need to fix the reason of described behavior instead of removing a block.